scripts/generate_tilesets.py

```diff
@@ -438,8 +438,13 @@ def main(cfg_file_path):
             logger.critical(e)
             sys.exit(1)
 
-        GT_tiles_gdf = gpd.sjoin(aoi_tiles_gdf, gt_labels_gdf, how='inner', predicate='intersects')
+        # split labels by tiles 
+        clipped_labels_gdf = misc.clip_labels(gt_labels_gdf, aoi_tiles_gdf, fact=0.9999)  
+        clipped_labels_gdf = clipped_labels_gdf.explode(ignore_index=True).drop(columns=['title','tile_id','x','y','z'])
```

Comment:
I don't have the "title" attribute in the project of the border points. It means it is not mandatory in the code.
In addition, if you have special columns, they are not dropped and duplicated with the suffixes "_left" and "_right" at line 446and then, the line 255 does not work anymore.
Wouldn't it be more versatile and give the same result to write: `clipped_labels_gdf = clipped_label_gdf.explode(ignore_index=True)[gt_labels_gdf.columns]`?